### ISSUE TITLE
Trained model

### ISSUE THREAD

**Issue by fanlyu** [Cyanogenoid/vqa-counting#5]
Can you share your trained model?

**Comment by ZhuFengdaaa** [comment]
+1

**Comment by Cyanogenoid** [comment]
I should have one model trained on the training set only and another on train+val ready in about 8--9 hours, assuming PyTorch 0.4 didn't break anything major (looks good so far). I need to train from scratch because the models that produced the results in the paper are pre-refactor and open-sourcing, so the weights are in a different format.

**Comment by Cyanogenoid** [comment]
A model trained on train and another model trained on trainval are available here now: https://github.com/Cyanogenoid/vqa-counting/releases/tag/resources

**Comment by Cyanogenoid** [comment]
Note that I have just updated the `pretrained-on-trainval.pth` model. I have retrained it with a small learning rate tweak (seen in the trainval branch), and now it gets basically exactly the same performance as the test-dev results in the paper, instead of being slightly worse.
